server/base/src/main/java/org/apache/accumulo/server/ServerConstants.java

```diff
@@ -68,7 +72,7 @@
   public static final int LOGGING_TO_HDFS = 4;
   public static final BitSet CAN_UPGRADE = new BitSet();
   static {
-    for (int i : new int[] {DATA_VERSION, MOVE_TO_ROOT_TABLE, MOVE_DELETE_MARKERS, LOGGING_TO_HDFS}) {
+    for (int i : new int[] {DATA_VERSION, MOVE_TO_REPLICATION_TABLE, MOVE_TO_ROOT_TABLE, MOVE_DELETE_MARKERS, LOGGING_TO_HDFS}) {
```

Comment:
I added 1.7 to the set of release we can upgrade from.  Probably need to remove some (like everyone that is not tested).   I think we should test the upgrade from 1.6 to 1.8 and remove all other releases.  Thoughts?  I can take a wack at testing 1.6 to 1.8 upgrade testing, unless someone else wants to do it.